client/me/concierge/shared/no-available-times.js

```diff
@@ -26,24 +27,58 @@ class NoAvailableTimes extends Component {
 				<PrimaryHeader isWhiteGlove={ isWhiteGlove } />
 				<Card>
 					<h2 className="shared__no-available-times-heading">
-						{ translate( 'Sorry, there are no sessions available' ) }
+						{ translate( 'Sorry, all upcoming sessions are full' ) }
 					</h2>
 					{ isWhiteGlove && (
 						<>
-							We schedule one-on-one sessions up to 24 hours in advance and all upcoming sessions
-							are full. Please check back later or{ ' ' }
-							<a href="https://wordpress.com/help/contact">contact us in Live Chat</a>.
+							We add new sessions daily, so please check back soon for more options. In the
+							meantime, consider attending one of our expert webinars on a wide variety of topics
+							designed to help you build and grow your site.{ ' ' }
+							<ExternalLinkWithTracking
+								icon={ false }
+								href="https://wordpress.com/webinars/"
+								onClick={ () => {} }
+								tracksEventName="calypso_concierge_book_view_webinars"
+							>
+								{ translate( 'View webinars.' ) }
+							</ExternalLinkWithTracking>
+							or
+							<ExternalLinkWithTracking
+								icon={ false }
+								href="https://wordpress.com/help/contact"
+								onClick={ () => {} }
+								tracksEventName="calypso_concierge_book_view_contact_us"
+							>
+								contact us in Live Chat
+							</ExternalLinkWithTracking>
+							.
+						</>
+					) }
+					{ ! isWhiteGlove && (
+						<>
+							{ translate(
+								'We add new sessions daily, so please check back soon for more options. In the meantime, consider attending one of our expert webinars on a wide variety of topics designed to help you build and grow your site.'
+							) }{ ' ' }
+							<ExternalLinkWithTracking
+								icon={ false }
+								href="https://wordpress.com/webinars/"
+								onClick={ () => {} }
+								tracksEventName="calypso_concierge_book_view_webinars"
+							>
+								{ translate( 'View webinars.' ) }
+							</ExternalLinkWithTracking>
+							{ translate( ' or ' ) }
```

Comment:
It's preferred to not break lines in translate() calls. Each string in a `translate()` call appears as its own word for translators, so they might lose context of the overall sentence. This is how I would rewrite this:

```
translate(
    'We add new sessions daily, so please check back soon for more options. In the meantime, consider attending one of our expert webinars on a wide variety of topics designed to help you build and grow your site. {{externalLink1}}View webinars{{/externalLink1}} or {{externalLink2}}contact us in Live Chat{{/externalLink2}}.',
    {
        components: {
            externalLink1: (
                                   <ExternalLinkWithTracking 
								       icon={ false }
								       href="https://wordpress.com/webinars/"
								       tracksEventName="calypso_concierge_book_view_webinars"
                                   />
                               ),
             externalLink2: ( ... )
        }
    }
)
```

Check this [example](https://github.com/Automattic/wp-calypso/blob/31ea12c6617e6298a96402fbc3b0269381de4e8c/client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx#L23).